data/data/bootstrap/baremetal/files/etc/containers/systemd/metal3-baremetal-operator.container.template

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Metal3 deployment service
+BindsTo=ironic-volume.service
+Requires=build-metal3-env.service wait-iptables-init.service image-customization.service build-ironic-env.service
+Wants=network-online.target ironic-httpd.service ironic-inspector.service ironic-ramdisk-logs.service
+After=network-online.target ironic-volume.service build-ironic-env.service wait-iptables-init.service provisioning-interface.service ironic-dnsmasq.service image-customization.service
+
+[Container]
+ContainerName=baremetal-operator
+Image=$METAL3_BAREMETAL_OPERATOR_IMAGE
+Exec=/baremetal-operator -build-preprov-image
+Network=host
+Volume=${AUTH_DIR}:/auth:z,ro
+Volume=/opt/openshift:/opt/openshift:z,ro
+Volume=ironic.volume:/shared:z
+Environment="XDG_RUNTIME_DIR=/run/user/${UID}"
+Environment="KUBECONFIG=/opt/openshift/auth/kubeconfig-loopback"
+Environment="DEPLOY_ISO=/shared/html/images/ironic-python-agent.iso"
+Environment="DEPLOY_KERNEL_URL=file://shared/html/images/ironic-python-agent.kernel"
+Environment="IRONIC_RAMDISK_SSH_KEY=${IRONIC_RAMDISK_SSH_KEY}"
+Environment="PROVISIONING_INTERFACE=${PROVISIONING_INTERFACE}"
+Environment="OS_CONDUCTOR__HEARTBEAT_TIMEOUT=120"
+Environment="IRONIC_HTPASSWD=${IRONIC_HTPASSWD}"
+Environment="INSPECTOR_HTPASSWD=${IRONIC_HTPASSWD}"
+Environment="IRONIC_KERNEL_PARAMS=${IRONIC_KERNEL_PARAMS}"
+Environment="HTTP_PORT=${HTTP_PORT}"
+Environment="IRONIC_INSPECTOR_ENDPOINT=${IRONIC_INSPECTOR_ENDPOINT}"
```

Comment:
nit: Inspector is no longer used directly

Reply:
Done